.github/workflows/integration.yml

```diff
@@ -81,3 +81,42 @@ jobs:
           CI: true
           VITE_API_KEY: ${{ secrets.VITE_API_KEY }}
 
+  deploy:
```

Comment:
This job shouldn't run on PR until we can commission preview deployment URLs on-demand. Running on PR will lead to red code stamping over our production instance